app/configurations/config.hb.js

```diff
@@ -126,6 +126,18 @@ export default configMerger(walttiConfig, {
           en: 'https://www.regioradstuttgart.de/',
         },
       },
+      "uri-demo": {
```

Comment:
This one can be removed as it was only to get a station that has a `rentalUriWeb`.

Reply:
Reverted.